x-pack/plugins/data_visualizer/public/application/data_comparison/data_comparison_view.tsx

```diff
@@ -0,0 +1,620 @@
+                }
+              )}
+            />
+          </div>
+        );
+      },
+    },
+    {
+      field: 'productionHistogram',
+      name: 'Production distribution',
+      'data-test-subj': 'mlDataComparisonOverviewTableDataComparisonDisbutrionChart',
+      sortable: false,
+      render: (productionDistribution: Feature['productionHistogram'], item) => {
+        return (
+          <div css={{ width: 100, height: 40 }}>
+            <DataComparisonDisbutrionChart
+              fieldType={item.fieldType}
+              data={productionDistribution}
+              color={colors.productionColor}
+              name={i18n.translate(
+                'xpack.dataVisualizer.dataComparison.dataComparisonDistributionLabel',
+                {
+                  defaultMessage: '{label} distribution',
+                  values: { label: PRODUCTION_LABEL },
+                }
+              )}
+            />
+          </div>
+        );
+      },
+    },
+    {
+      field: 'comparisonDistribution',
+      name: 'Comparison',
+      'data-test-subj': 'mlDataComparisonOverviewTableDataComparisonDisbutrionChart',
+      sortable: false,
+      render: (comparisonDistribution: Feature['comparisonDistribution'], item) => {
+        return (
+          <div css={{ width: 100, height: 40 }}>
+            <OverlapDistributionComparison
+              fieldName={item.featureName}
+              fieldType={item.fieldType}
+              data={comparisonDistribution}
+              colors={colors}
+            />
+          </div>
+        );
+      },
+    },
+  ];
+
+  const getRowProps = (item: Feature) => {
+    return {
+      'data-test-subj': `mlDataComparisonOverviewTableRow row-${item.featureName}`,
+      className: 'mlDataComparisonOverviewTableRow',
+      onClick: () => {},
+    };
+  };
+
+  const getCellProps = (item: Feature, column: EuiTableFieldDataColumnType<Feature>) => {
+    const { field } = column;
+    return {
+      className: 'mlDataComparisonOverviewTableCell',
+      'data-test-subj': `mlDataComparisonOverviewTableCell row-${item.featureName}-column-${String(
+        field
+      )}`,
+      textOnly: true,
+    };
+  };
+
+  const toggleDetails = (item: Feature) => {
+    const itemIdToExpandedRowMapValues = { ...itemIdToExpandedRowMap };
+
+    if (itemIdToExpandedRowMapValues[item.featureName]) {
+      delete itemIdToExpandedRowMapValues[item.featureName];
+    } else {
+      const { featureName, comparisonDistribution } = item;
+      itemIdToExpandedRowMapValues[item.featureName] = (
+        <div css={{ width: '100%', height: 200 }}>
+          <DataComparisonChart
+            featureName={featureName}
+            fieldType={item.fieldType}
+            data={comparisonDistribution}
+            colors={colors}
+          />
+        </div>
+      );
+    }
+    setItemIdToExpandedRowMap(itemIdToExpandedRowMapValues);
+  };
+
+  return (
+    <EuiInMemoryTable<Feature>
```

Comment:
Is the paging working correctly here? I only ever see 1 page of data e.g. page size of 10 - only see the first 10 rows.

Reply:
Fixed here [`f9a2527` (#161365)](https://github.com/elastic/kibana/pull/161365/commits/f9a2527eda999a5b504ffd9faf137b3e8e8be424)